src/components/Composer/index.js

```diff
@@ -56,13 +56,19 @@ const replaceCaret = (el) => {
 	}
 };
 
+
+let inputLock = false;
```

Comment:
We could transform this into a state instead a global variable.

Reply:
@sctEdwin You marked this comment as resolved but you didn't change it. Why?

Reply:
@rentaobecker
Sorry I am new to this, did my changes address the issue?